beanstalk_worker/worker_start_scan.py

```diff
@@ -110,7 +110,8 @@ def run(self):
 
         # TODO: Figure out why random tag (with date) is coming
         # image_under_test = ":".join(self.job_info.get("name").split(":")[:-1])
-        image_under_test = self.job_info.get("name")
+        image_under_test = self.job_info.get("name") + ":" \
+            + self.job_info.get("tag")
```

Comment:
@bamachrn @navidshaikh need your comments on this line. I'm worried it'd do some harm and not always work as expected! Because there are instances where we specify tag info in the name key itself.